src/commands/sbom/README.md

```diff
@@ -1,36 +1,37 @@
 # SBOM command
 
-<div class="alert alert-warning"><strong>Warning:</strong> The <code>SBOM upload</code> command is in beta. It requires you to set <code>DD_BETA_COMMANDS_ENABLED=1</code>, and should not be used in production.</div>
-
 This command lets you upload SBOM files to the Datadog intake endpoint.
 
 
 ## Supported Formats
 
  - CycloneDX 1.4
+ - CycloneDX 1.5
 
 ## Usage
 
 ```bash
-DD_BETA_COMMANDS_ENABLED=1 datadog-ci sbom upload --service <my-service> <path/to/sbom.json>
+datadog-ci sbom upload <path/to/sbom.json>
 ```
 
+### Optional arguments
+
+- `--service` should be set to the name of the service you're uploading SBOM reports from.
+- `--env` is a string that represents the environment in which you want your tests to appear.
+
 ### Environment variables
 
 The following environment variables must be defined:
 
  - `DD_SITE`: the [Datadog site](https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site)
  - `DD_APP_KEY`: the App key to use
  - `DD_API_KEY`: the API key to use
- - `DD_SERVICE`: the Datadog service you use (if `--service` not specified)
-
 
 ## Development
 
 When developing software, you can try with the following command:
 
-```bash
-DD_BETA_COMMANDS_ENABLED=1 yarn launch sbom upload --service <your-service> --env <your-environment> /path/to/sbom.json
+```bashyarn launch sbom upload --service <your-service> --env <your-environment> /path/to/sbom.json
```

Comment:
I don't think this is rendered correctly
```suggestion
```bash
yarn launch sbom upload --service <your-service> --env <your-environment> /path/to/sbom.json
```